src/styles/_animation.scss

```diff
@@ -0,0 +1,7 @@
+.anim__minutes--hand {
```

Comment:
split files by semantic(file by BEM block)
keyframes file is ok, by better to name keyframes as animations